packages/reparse-lwt-unix/reparse-lwt-unix.3.0.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Reparse lwt-unix based input support"
+description: "Reparse lwt-unix based input support"
```

Comment:
The `synopsis` takes precedence over `description`. For next time could you elaborate a little more in the description?

Reply:
or remove description, this would also be totally fine of course

Reply:
sure.